Add the ability to connect to existed Chrome (#456)
ruby/debug: README.md

Also `open chrome` command works like `open vscode`.

> Debugger tries to open Chrome automatically in some platforms and can attach to opened one from the second time unless you close it.

Reviewer: "close it." -> "close the chrome browser".

Also `open chrome` command works like `open vscode`.

Debugger tries to open Chrome browser automatically in some platforms and can attach to opened one from the second time unless you close the Chrome browser.

```
$ rdbg target.rb --open=chrome
DEBUGGER: Debugger can attach via TCP/IP (127.0.0.1:54004)
DEBUGGER: wait for debugger connection...
Launching: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --remote-debugging-port=0 --no-first-run --no-default-browser-check --user-data-dir=/var/folders/kv/w1k6nh1x5fl7vx47b2pd005w0000gn/T/d20220102-83390-huc0yn

> DEBUGGER: Run the following command if you want to debug again with opened Chrome browser's process:

Reviewer: ```suggestion
DEBUGGER: Use a following environment variable if you want to use the opened browser again:
```

    RUBY_DEBUG_REMOTE_DEBUGGING_PORT=54005/devtools/browser/658084a0-3d64-4049-ae2b-5e663c19685e rdbg --open=chrome target.rb

> *NOTE*: DO NOT use `RUBY_DEBUG_REMOTE_DEBUGGING_PORT` without opened one.

Reviewer: ```suggestion
NOTE: DO NOT use `RUBY_DEBUG_CRHOME_REMOTE_DEBUGGING_PORT` after closing the opened browser.
```

Reviewer 2: `RUBY_DEBUG_CHROME_REMOTE_DEBUGGING_PORT`
